Dockerfile

```diff
@@ -20,4 +20,4 @@ COPY --from=build /Silk/out .
 
 RUN chmod +x ./Silk.Core
 
-CMD ["./Silk.Core"]
+CMD ["./Silk.Core"]
```

Comment:
🤔 Github???

Reply:
Don't feel bad; I got a merge conflict on an entire file yesterday. A file created on the branch I was PRing onto master at that.